PWGCF/FemtoDream/FemtoDreamDetaDphiStar.h

```diff
@@ -153,7 +153,7 @@ class FemtoDreamDetaDphiStar
 
     float phi0 = part.phi();
     // Start: Get the charge from cutcontainer using masks
-    float charge;
+    float charge{};
```

Comment:
There is no reason for a float. I could just use `float charge = 0` (it is the same).
In other places I needed a general constructor because it wase a template variable.
I can change it, but in the end it is the same.

Reply:
I see. I was not familiar with the `{}` construction